server/api/discrepancies.py

```diff
@@ -187,6 +197,15 @@ def show_batch_comparison_discrepancies_by_jurisdiction(
     }
 
 
+# Check if any value is non-zero for the audited batch result, indicating the batch has actually been audited
+def is_audited_batch_finalized(audited_batch_result: Dict[str, Dict[str, int]]) -> bool:
+    return any(
+        value != 0
+        for contest in audited_batch_result.values()
+        for value in contest.values()
+    )
```

Comment:
`sampled_batch_results` was originally written to be used only in cases where all the batches had been audited (p-value calculation, sample sizes calculation), so it assumes that's the case. Now that we're using it during the round, that assumption breaks. I think we should fix this bug there by filtering out batches that haven't been audited yet (i.e. have no recorded tally sheets).

The solution you have here will work in all cases except the case when a batch is actually audited as having 0 votes for each candidate. This is probably never going to happen, but still would rather use a fully safe solution.

Reply:
Thanks for that context, I've [added a filter](https://github.com/votingworks/arlo/pull/2095/commits/f037d934dc9e6d3310f5095c6eea5827b7364971) to `sampled_batch_results` instead that seems to work as intended. 

![Screenshot 2024-12-16 at 1 35 22 PM](https://github.com/user-attachments/assets/dbcd92e7-7125-4954-a868-b33ee28781bf)